Sources/SFSafeSymbols/ImageExtension.swift

```diff
@@ -0,0 +1,14 @@
+import SwiftUI
+
+@available(iOS 13.0, *)
+public extension Image {
+
+    /// Creates a instance of `Image` with a `UIImage` object that has its rendering mode set to `.alwaysTemplate`.
+    ///
+    /// - systemSymbol: The `SFSymbol` describing this image.
+    @available(iOS 13.0, *)
+    init(systemSymbol: SFSymbol) {
```

Comment:
Is there a specific reason there aren't initializers for `UIImage.Configuration` and `UITraitCollection`? If no, would you mind adding those too (Also to the README)?